Makefile

```diff
@@ -1,4 +1,5 @@
-OPEN_DOCS?="-open"
+VERSION   ?= 1.0.0-beta2
```

Comment:
An other idea instead of having it in the Makefile would be to feed it into a `.version` file to make it better accessible to other scripts. But this will do for now.

Reply:
That could be added in the future by changing the assignment to `$(shell cat .version)`.